apps/arweave/src/ar_block_pre_validator.erl

```diff
@@ -802,7 +802,7 @@ priority(B, Peer) ->
 	{B#block.height, get_peer_score(Peer)}.
 
 get_peer_score(Peer) ->
-	get_peer_score(Peer, ar_peers:get_peers(lifetime), 0).
```

Comment:
This *might* need to stay as `lifetime`. Or rather: I think we need to differentiate between 3 cases (so far):

1. peer list and ranking that we use to prioritize block validation and gossiping. Currently `lifetime`
2. peer list and ranking that we use for syncing chunks. Currently `current` (via `ar_data_discovery`)
3. Peer list and ranking that we use for downloading headers. Currently `lifetime`

My gut feel is that we should probably do something like:
1. keep at `lifetime` for now
2. Keep at `current` for now *or* update to something like `days, 30`
3. `days, 30`


@ldmberman what do you think? 

I think there's an argument to be made that all values could purge peers after X days, however the `lifetime` ranking is different than the `current` ranking and I don't think we want to change that yet. `lifetime` considers more data point and so is harder to game which is important for stuff like gossiping and validation order. `current` biases towards recent data and so is important for performance-critical behavior like syncing chunks and header data

But I'm not sure what's best in general. One option is we have 2 thresholds - e.g.
`lifetime` does ranking based on all data points and purges after 90 days 
`current` does the EMA ranking as it does now (biasing towards recent data points) and purges after 30 days.

In this way we still only have two peer lists: lifetime and current.

In all cases we probably want the time window to be a `-define` constant rather than hardcoded value.